emoji/src/components/EmojiRow.js

```diff
@@ -0,0 +1,11 @@
+import React, { Component } from 'react';
+
+class EmojiRow extends Component {
+    render() {
```

Comment:
If your react component has only `render` method, write it as a pure function: `const EmojiRow = () => <h1>Hello</h1> ` . Just one line would replace the whole code. Furthermore, functions are better in performance